lib/membrane_timescaledb_reporter/reporter.ex

```diff
@@ -0,0 +1,75 @@
+defmodule Membrane.Telemetry.TimescaleDB.Reporter do
```

Comment:
It might be out of the scope of this PR, but have you seen https://hexdocs.pm/telemetry_metrics/Telemetry.Metrics.html ? Have you considered writing a more general reporter according to this: https://hexdocs.pm/telemetry_metrics/writing_reporters.html?